### ISSUE TITLE
Verified Skill: Backend API did not update when member win a related challenge

### ISSUE THREAD

**Issue by luizrrodrigues** [topcoder-platform/community-app#2376]
This PR is just to **Frontend**, **Backend API** already returning **CHALLENGE** and **USER_ENTERED** in `source` of `skills` data.

QA Question:
- Backend API already add **CHALLENGE** source to skill when member win a Challenge?

Looks like not working, check this profile below, you can see **QA** in skill WITHOUT **Verified** mark:
https://beta-community-app.topcoder.com/members/sdgun   
![image](https://user-images.githubusercontent.com/7029641/58534481-d09afc00-81c1-11e9-9e0e-0d7ade6ff432.png)

But check this Challenge (member on this Challenge), have **QA** in tech:
https://beta-community-app.topcoder.com/challenges/30069360
![image](https://user-images.githubusercontent.com/7029641/58534505-e4466280-81c1-11e9-9198-63194dab1c36.png)

So, if member win Challenge with **QA** tag, why API doesn't return **CHALLENGE** to QA skill?

**Comment by luizrrodrigues** [comment]
@urwithat Please me know when you have a chance to check this one.

Thanks.

**Comment by Oanh-and-only-Oanh** [comment]
@luizrrodrigues, is this only the case if the member originally added the skill himself but later he wins a challenge with that same skill tag? The data loader is not updating that new SOURCE to be CHALLENGE. Is this correct?

**Comment by nithyaasworld** [comment]
> @luizrrodrigues, is this only the case if the member originally added the skill himself but later he wins a challenge with that same skill tag? The data loader is not updating that new SOURCE to be CHALLENGE. Is this correct?

Yes, correct @Oanh-and-only-Oanh . I am not sure if thats the only the case. But that was the only case I was able to test with the existing data.

I have few questions regarding the function. 

1. Source will be updated to Challenge when user wins the first place? or any winning place (second or third place) is also considered?
2. If the skills of the Challenge which the user has won, is not already present in his Profile, then system will add the skill and then set its source to Challenge? 

Once you confirm the answers, I will have to check with @luizrrodrigues on how to verify them. 

Thank You!

**Comment by Oanh-and-only-Oanh** [comment]
@urwithat, can you help any Nithya's questions above? 

@nithyaasworld @luizrrodrigues From a use case, we want the Verified Skill symbol to appear for any winning place.

**Comment by urwithat** [comment]
@Oanh-and-only-Oanh - I'm not aware of this functionality

**Comment by Oanh-and-only-Oanh** [comment]
@RishiRajSahu, can you help with this?

**Comment by RishiRajSahu** [comment]
@Oanh-and-only-Oanh Haven't touched skills, but for investigation @luizrrodrigues could you check the api endpoint(s) which are returning those data and share them here on the ticket, so that we can reach to right owner ?
